content/docs/guide-development.md

```diff
@@ -117,18 +117,14 @@ https://renkulab.io/
 
 TODO: explain how to run server
 
-### Developing plugins for Renku
-
-Renku provides a number of hooks that can be used to add additional customized metadata. It provides also the possibility to add CLI commands.
-
-[https://renku.readthedocs.io/en/stable/renku-python/docs/reference/plugins.html](https://renku.readthedocs.io/en/stable/renku-python/docs/reference/plugins.html)
+### Developing service in Renku
```

Comment:
this entire guide is about developing service in renku, see above in this document. But maybe we can edit a bit more ourselves, it's easier to see as users.